examples/language-modeling/run_lora_clm.py

```diff
@@ -92,6 +92,10 @@ class ModelArguments:
         default=None,
         metadata={"help": "Where do you want to store the pretrained models downloaded from huggingface.co"},
     )
+    auth_token: Optional[str] = field(
+        default=None,
+        metadata={"help": "auth token for private models"},
+    )
```

Comment:
Let's call it `token` as in other examples: https://github.com/huggingface/optimum-habana/blob/3a13a0141d7417b709fe22fd177228c834117bbf/examples/language-modeling/run_clm.py#L122

Reply:
Will update in next patch.